assets/airports/lrop.json

```diff
@@ -0,0 +1,674 @@
+        "TEVRO": ["N44d20m47.081", "E025d30m07.333"],
+        "TOBAG": ["N44d22m37.978", "E026d42m09.949"],
+        "TOSVI": ["N44d45m14.010", "E025d09m40.719"],
+        "TUTIX": ["N44d14m24.422", "E025d31m07.611"],
+        "UVALU": ["N44d42m38.891", "E025d26m38.864"],
+        "BOGVI": ["N44d42m03.093", "E026d38m07.810"],
+        "DILIM": ["N44d37m13.027", "E026d50m37.704"],
+        "DONOX": ["N44d11m05.174", "E025d54m23.878"],
+        "ESUXU": ["N44d27m30.793", "E026d26m06.479"],
+        "IDARU": ["N44d38m24.506", "E026d58m54.475"],
+        "ITVUK": ["N44d28m00.963", "E026d33m03.413"],
+        "IVDEK": ["N44d28m30.711", "E026d40m00.469"],
+        
+        "SOKRU": ["N44d59m34.136", "E025d19m49.048"],
+        "OKAGO": ["N44d47m48.779", "E026d34m19.568"],
+
+
+        "GOMDI": ["N44d51m54.285", "E025d56m39.717"],
+        "TABAV": ["N44d53m22.808", "E026d58m01.012"],
+        "USIRO": ["N44d14m30.397", "E025d45m52.599"],
+        "EKSUN": ["N44d55m13.731", "E026d17m08.121"],
+        "ELKAV": ["N44d55m43.219", "E026d51m38.107"],
+        "ETUXA": ["N44d49m34.939", "E026d17m57.475"],
+        "OP881": ["N44d35m05.531", "E026d11m36.734"],
+        "OP882": ["N44d34m24.940", "E026d11m42.684"],
+        "OP883": ["N44d35m42.748", "E026d19m57.990"],
+        "OP884": ["N44d35m02.161", "E026d20m03.842"],
+        "OP885": ["N44d37m04.432", "E026d38m48.993"],
+        "OP886": ["N44d36m19.747", "E026d37m56.725"],
+        "OP887": ["N44d37m54.519", "E026d50m45.222"],
+        "OP888": ["N44d37m06.677", "E026d49m05.656"],
+        "OP889": ["N44d21m59.577", "E026d21m56.214"],
+        "OP890": ["N44d46m02.123", "E026d10m00.159"],
+        "OP891": ["N44d49m16.707", "E026d02m38.662"],
+        
+        "ENIMA": ["N45d04m46.00","E24d54m09.00"],
+        "PELUR": ["N45d17m12.00","E25d29m18.00"],
+        "LAPKA": ["N45d17m34.00","E26d30m33.00"],
+        "URELA": ["N45d29m48.00","E26d33m40.00"],
+        "RIVOS": ["N44d39m21.00","E27d39m17.00"],
+        "DIRAL": ["N44d30m39.00","E27d33m15.00"],
+        "KOMAN": ["N43d59m00.00","E26d13m00.00"],
+        "ARGES": ["N44d04m56.00","E26d49m36.00"],
+        "RASUB": ["N43d58m44.00","E25d25m06.00"],
+        "ORTIP": ["N43d58m40.00","E25d19m59.00"],
+        "ELVAB": ["N44d09m52.00","E25d05m27.00"],
+        "VIKBI": ["N44d15m31.00","E24d39m21.00"],
+        "UBOGU": ["N44d45m47.00","E24d40m06.00"],
+        "VAMON": ["N44d23m58.00","E24d40m47.00"],
+        "BULEN": ["N43d45m00.00","E25d49m00.00"],
+        "PEMOK": ["N44d21m52.90","E24d20m54.50"],
+        "LAMIT": ["N45d06m14.00","E23d22m29.00"],
+        "KODRU": ["N46d11m32.00","E28d07m26.00"],
+        "FOCSA": ["N45d59m41.00","E26d41m23.00"],
+        "SOMOV": ["N43d42m00.00","E24d51m00.00"],
+
+        "BIVBU": ["N44d55m14.00","E24d19m54.00"] 
+
+    },
+    "runways": [
+        {
+            "name":        ["08R", "26L"],
+            "name_offset": [[0, 0], [0, 0]],
+            "length": 3.5,
```

Comment:
The `length` attribute is deprecated as it is now calculated from the start and end coordinates of the runway